src/page/Variant.css

```diff
@@ -23,3 +23,9 @@
     display: flex;
     justify-content: center;
 }
+
+/* hide mutation mapper meta column */
+/* we cannot disable the whole meta column because we need the computed data */
+.gn-mutation-mapper .mutationMapperMetaColumn {
+    display: none;
+}
```

Comment:
This doesn't seem ideal. There might be a better way to trigger the computation of those values instead of rendering something we don't intend to show.

Reply:
can't figure out a good way to trigger the computation... Add a todo there for now